src/main/java/com/google/api/codegen/transformer/go/GoGapicSurfaceTransformer.java

```diff
@@ -188,6 +188,13 @@ private StaticLangXExampleView generateExample(SurfaceTransformerContext context
         generateApiMethods(
             context, context.getSupportedMethods(), Collections.<String>emptyList()));
 
+    // Examples are different from the API. In particular, we use short declaration
+    // and so we omit most type names. We only need
+    //   context, to initialize the client
+    //   the VKit generated library, that's what the sample is for
+    //   the input types of the methods, to initialize the requests
```

Comment:
Put numbers on these comments and capitalize the first word of each line - as it is now, it looks like a weirdly formatted run-on sentence.